components/Dialogs/examples/DialogsTypicalUseExampleViewController.m

```diff
@@ -35,14 +34,16 @@ @implementation DialogsTypicalUseExampleViewController
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  if (!self.containerScheme) {
-    self.containerScheme = [[MDCContainerScheme alloc] init];
-  }
-  if (!self.colorScheme) {
-    self.colorScheme =
+  if (self.containerScheme == nil) {
```

Comment:
Per my previous comment - this is done right.
It looks like there are multiple ways to instantiate the color scheme in different examples.
Let's be consistent and make sure all examples follow this one.

Reply:
This is actually a little wrong, if `self.containerScheme` is not nil it's `colorScheme` can still be nil.